dvc/scm/git/__init__.py

```diff
@@ -253,7 +256,9 @@ def remind_to_track(self):
             "\n"
             "To track the changes with git, run:\n"
             "\n"
-            "\tgit add {files}".format(files=" ".join(self.files_to_track))
+            "\tgit add {files}".format(
+                files=" ".join(set(self.files_to_track))
```

Comment:
so maybe we should actually make self.files_to_track a set?

Reply:
@efiop, can you review this again, please?